recipes/nn.rb

```diff
@@ -172,4 +172,50 @@
     EOF
   end
 
+
+  ts = Time.new.strftime("%Y_%m_%d_%H_%M")
+
```

Comment:
@jimdowling Can you wrap this in an if so that it's executed only on the "first" namenode? Otherwise you can still get concurrent executions in case of multi-NN, and you might end up creating a larger DATAFILE than the actual drive

Reply:
This is already wrapped for the first namenode - it is inside this condition:
if my_ip.eql? node['hops']['nn']['private_ips'][0]